Products/ZenModel/migrate/addPrefixToTriggerRule.py

```diff
@@ -0,0 +1,50 @@
+##############################################################################
+# 
+# Copyright (C) Zenoss, Inc. 2015, all rights reserved.
+# 
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+# 
+##############################################################################
+
+
+__doc__='''
+
+Add 'evt.' prefix to trigger rules and custom fields definition
+in order to make possible evaluate trigger rules with them.
+ 
+'''
+
+import Migrate
+from zenoss.protocols.jsonformat import from_dict
+from zenoss.protocols.protobufs.zep_pb2 import EventDetailItemSet
+import zenoss.protocols.protobufs.zep_pb2 as zep
+from Products.Zuul import getFacade
+
+class addPrefixToTriggerRule(Migrate.Step):
+    version = Migrate.Version(5, 0, 70)
```

Comment:
This should be Migrate.Version(5, 0, 3) - since we are working on releasing 5.0.3